ghedesigner/ghe/bisection_zd_search.py

```diff
@@ -2,10 +2,11 @@
 from pygfunction.boreholes import Borehole
 
 from ghedesigner.enums import FlowConfigType, TimestepType
-from ghedesigner.ghe.bisection_1d_search import Bisection1D
-from ghedesigner.ghe.simulation import SimulationParameters
 from ghedesigner.media import GHEFluid, Grout, Pipe, Soil
 
+from .bisection_1d_search import Bisection1D
+from .simulation import SimulationParameters
+
```

Comment:
@axelstudios when do we know which import style to use?

Reply:
OK, if it's all the same, I'd prefer to just stick with absolute imports where possible.